### PR TITLE
virttest/qemu_monitor.py: Add cmd "qom-set" and cmd "qom-get".

### PR DESCRIPTION
**Pull request by CongSmile**
Signed-off-by: Cong Li <coli@redhat.com>

id: 1231037

### CODE REVIEW

File: virttest/qemu_monitor.py [hunk 1]
```diff
@@ -2011,3 +2011,30 @@ def change_media(self, device, target):
         self.verify_supported_cmd(cmd)
         args = {"device": device, "target": target}
         return self.cmd(cmd, args)
+
+    def qom_set(self, path, properties, value):
```
Comment (xutian) [inline virttest/qemu_monitor.py]: s/properties/property/g

**Comment by CongSmile** [comment]
@xutian 

Just want to avoid the key word in the code.
But it's fine to use "property", could you help review it again?

Thanks.

**Comment by xutian** [comment]
qproperty and qvlaue will better :)

**Comment by CongSmile** [comment]
@xutian 

Done, could you review it again?

**Comment by xutian** [comment]
looks fine, merged.

**Pull request merged**